components/script/dom/webidls/HTMLInputElement.webidl

```diff
@@ -41,7 +41,7 @@ interface HTMLInputElement : HTMLElement {
   //         attribute DOMString step;
            attribute DOMString type;
            attribute DOMString defaultValue;
-  [TreatNullAs=EmptyString] attribute DOMString value;
+  [TreatNullAs=EmptyString, SetterThrows] attribute DOMString value;
```

Comment:
nit: Move the `attribute DOMString value;` to a new line and preserve its spacing.